api/changes/fabo_fix-node-subscriptions

```diff
@@ -0,0 +1 @@
+[Fixed] Fix new polkadot node subscriptions @faboweb
```

Comment:
Why my change is now gone?

Reply:
which change?

Reply:
The one for "return to old polkadot node subscription".

When a change gets rebased by a new bigger change in that file Simsala automatically deletes it or how does it work?

Reply:
no, I reverted it